pkg/apis/resource/types.go

```diff
@@ -954,6 +990,42 @@ type DeviceRequestAllocationResult struct {
 	// +optional
 	// +featureGate=DRAAdminAccess
 	AdminAccess *bool
+
+	// UsageRestrictedToNode indicates if the usage of an allocation involving this device
+	// has to be limited to exactly the node that was chosen when allocating the claim.
+	//
+	// +optional
+	UsageRestrictedToNode *bool
+
+	// BindingConditions defines the conditions for proceeding with binding.
+	// All of these conditions must be set in the per-device status
+	// conditions with a value of True to proceed with binding the pod to the node
+	// while scheduling the pod.
+	// The maximum number of binding conditions is 4.
+	//
+	// +optional
+	// +listType=atomic
+	BindingConditions []string
```

Comment:
This needs to be in the status because it is tied to the devices in the allocation. The definition of those could have changed in the meantime, or the slice is unavailable.

+featureGate again.

Reply:
Yes. All new fields are feature-gated.

Reply:
You can resolve this thread.